### PR TITLE
fix: rpa contract amendments (CMC-1496)

### PR DESCRIPTION
**Pull request by sabahirfan**
### JIRA link (if applicable) ###
[CMC-1496](https://tools.hmcts.net/jira/browse/CMC-1496)

### Change description ###
RPA contract amendments as per discussion over the call with robotics team.

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/unspec/service/robotics/mapper/EventHistoryMapper.java [hunk 1]
```diff
@@ -267,6 +273,7 @@ private void buildMiscellaneousEvents(
                     .eventSequence(1)
                     .eventCode("999")
                     .dateReceived(caseData.getRespondent1ResponseDate().format(ISO_DATE))
+                    .eventDetailsText("Claimant has notified defendant")
```
Comment (ckalista) [inline src/main/java/uk/gov/hmcts/reform/unspec/service/robotics/mapper/EventHistoryMapper.java:276]: Are we missing a dot at the end here?

File: src/main/java/uk/gov/hmcts/reform/unspec/service/robotics/mapper/EventHistoryMapper.java [hunk 2]
```diff
@@ -267,6 +273,7 @@ private void buildMiscellaneousEvents(
...
                     .eventDetails(EventDetails.builder()
                                       .miscText("Claimant has notified defendant")
```
Comment (ckalista) [inline src/main/java/uk/gov/hmcts/reform/unspec/service/robotics/mapper/EventHistoryMapper.java:278]: Are we missing a dot at the end here?

**Review by ckalista — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/unspec/service/robotics/mapper/EventHistoryMapper.java (hunk 1 above)
Comment (sabahirfan) [reply src/main/java/uk/gov/hmcts/reform/unspec/service/robotics/mapper/EventHistoryMapper.java:276]: will make it consistent in coming PR

Comment (sabahirfan) [reply src/main/java/uk/gov/hmcts/reform/unspec/service/robotics/mapper/EventHistoryMapper.java:276]: will make it consistent in coming PR

File: src/main/java/uk/gov/hmcts/reform/unspec/service/robotics/mapper/EventHistoryMapper.java (hunk 2 above)
Comment (sabahirfan) [reply src/main/java/uk/gov/hmcts/reform/unspec/service/robotics/mapper/EventHistoryMapper.java:278]: will make it consistent in coming PR

**Review by dharmendrak — APPROVED** [review]

**Pull request merged**